LandingPage.js

```diff
@@ -0,0 +1,69 @@
+import styles from './LandingPage.module.css';
+import classNames from 'classnames';
+
+function LandingPage() {
+    return (
+        <div>
+            <header>
+                <div className={styles.navBar}>
+                    <a href="#home">HOME</a>
+                    <a href="#signUp">SIGN UP</a>
+                    <a href="#logIn">LOG IN</a>
+                    <a href="#aboutUs">ABOUT US</a>
+                </div>
+            </header>
+            <h1>CAREER CONNECT</h1>
+            <div className={styles.mainContent}>
+                <section className={styles.container}>
+                    <div className={styles.content}>
+                        <h2>Loren Ipsum</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet, sed unum adversarium disputationi eu, aliquid conceptam mel ut. Vel modo nusquam ad, cu sea oratio legere ornatus. Quo ex ignota senserit, ei sea soluta fuisset. Sea ex impetus mediocrem intellegat. Sea ad enim nobis convenire, his brute velit ea, mel an clita reprimique.

```

Comment:
Since you're taking on the task of creating the landing page we should try to implement to its finished product. This looks like you put temporary text for now but please change this to the actual content for the landing page of career connect. 

Here is the mockup for the landing page that Jason created for us to have a idea of how the landing page may look but you can get as creative as you want: https://www.figma.com/design/TzYtjBdMGoKkkG2w1QyMwV/Career-Roadmap-%26-Skill-Swap?node-id=9-53&t=S0IXMADczwRD8ynl-0

Reply:
Can I change this once we start working more on the different features? I think we're trying to making it less like LinkedIn and I want to make sure I'm explaining the site's use properly